ansible/group_vars/all

```diff
@@ -268,6 +270,8 @@ linux:
 couchdb:
   version: 2.1
 
+limit_activation_payload: 1048576
```

Comment:
add a comment for describing this value's purpose.. payload request, response, includes metdata+logs+results?